teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<p>This paragraph can be reworded as the following to avoid some repetition and improve clarity. Some typos are also fixed here, so you can just copy paste if you agree.</p><p> </p><blockquote>In the modern drug discovery pipeline<strong>,</strong> determining the binding mode of an active molecule to a given protein target is of utmost importance. Such information can suggest novel chemical modifications to optimize interactions between ligand and protein, thus increasing the binding affinity. Molecular docking software can predict these binding modes by sampling possible conformations of a ligand inside the protein binding pocket (Fig. 1). A scoring function is thereby used to estimate the quality of each binding pose, which is commonly calculated with a variety of terms for different non-covalent molecular interactions; e.g., electrostatic and van der Waals energy (<a href="https://doi.org/10.1007/s12551-016-0247-1" target="_blank"><em>Biophy Rev</em> (2017), <strong>9</strong>, 91-102</a>). Also, the rise of artificial intelligence lead to the development of algorithms able to generate libraries with billions of theoretically synthesizable molecules (<a href="https://doi.org/10.1038/s41570-018-0066-y" target="_blank"><em>Nat Rev Chem</em> (2019), <strong>3</strong>, 119-128</a>). Molecular docking can probe these molecules before they were even synthesized and thus, accelerate the discovery of novel lead candidates.</blockquote><p> </p><p>---</p><p> </p><p>The list seems to be formatted weirdly, but that could be ReviewNB parsers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>